core/src/test/java/org/mule/util/ClassUtilsTestCase.java

```diff
@@ -352,6 +354,55 @@ public void setInheritedFieldValueWithoutRecurse() throws Exception
         ClassUtils.setFieldValue(new ExtendedHashBlob(1), "hash", 0, false);
     }
 
+    @Test
+    public void isInstance()
+    {
+        assertThat(ClassUtils.isInstance(String.class, null), is(false));
+        assertThat(ClassUtils.isInstance(String.class, ""), is(true));
+        assertThat(ClassUtils.isInstance(Fruit.class, new Apple()), is(true));
+        assertThat(ClassUtils.isInstance(Apple.class, new Kiwi()), is(false));
+        assertThat(ClassUtils.isInstance(int.class, 0), is(true));
+        assertThat(ClassUtils.isInstance(Integer.class, 0), is(true));
+        assertThat(ClassUtils.isInstance(Long.class, 0L), is(true));
+    }
+
+    @Test
+    public void isWrapperAndPrimitiveType()
```

Comment:
isWrapperAndPrimitiveType and isPrimitiveWrapper tests are basically testing Primitives class. We should test only our logic and I think that is already covered on the isInstance test